cloudformation/memsub-promotions-lambdas-cf.yaml

```diff
@@ -248,7 +285,36 @@ Resources:
       FunctionName: MembershipSub-PromoCode-View-Dynamo-to-Salesforce-PROD
       Handler: MembershipSub-PromoCode-View-Dynamo-to-Salesforce.handler
       MemorySize: 128
-      Role: !GetAtt [MembershipSubPromotionsDataRole, Arn]
+      Role: !GetAtt [MembershipSubPromoCodeViewETLRole, Arn]
       Runtime: nodejs6.10
       Timeout: 10
-      KmsKeyArn: !GetAtt SalesforceLambdaKMSKey.Arn
\ No newline at end of file
+      KmsKeyArn: !GetAtt SalesforceLambdaKMSKey.Arn
+  ScheduledBackupLambdaPROD:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        S3Bucket: gu-promotions-tool-dist
+        S3Key: membership/PROD/MembershipSub-Promotions-PromoCode-View/MembershipSub-Promotions-PromoCode-View.zip
+      Description: A Lambda function to schedule the backup of the Campaigns and Promotions table
+      FunctionName: MembershipSub-Promotions-Scheduled-Dynamo-Backup-PROD
+      Handler: MembershipSub-Promotions-Scheduled-Dynamo-Backup.handler
+      MemorySize: 128
+      Role: !GetAtt [MembershipSubPromotionsDataBackupLambdaRole, Arn]
+      Runtime: nodejs6.10
+      Timeout: 10
```

Comment:
This seems quite short - does this give us enough time to create the backups and delete all of the old ones?

Reply:
I guess after the initial run the delete job will be much faster...